e2e/loadbalancer_test.go

```diff
@@ -92,107 +105,75 @@ func TestLoadbalancerProxy(t *testing.T) {
 	}
 
 	fmt.Println("Creating Service")
-	err = e2eTest.tenantClient.Create(context.TODO(), svc)
+	err = e2eTest.tenantClient.Create(ctx, svc)
 	g.Expect(err).ShouldNot(HaveOccurred())
 
 	g.Eventually(func() string {
-		err = e2eTest.tenantClient.Get(context.TODO(), client.ObjectKeyFromObject(svc), svc)
+		err = e2eTest.tenantClient.Get(ctx, client.ObjectKeyFromObject(svc), svc)
 		if len(svc.Status.LoadBalancer.Ingress) == 0 {
 			return ""
 		}
 		return svc.Status.LoadBalancer.Ingress[0].Hostname
 	}, "5m", "5s").ShouldNot(BeEmpty())
 
 	// Cleanup
-	err = cleanUp(mirrorDeploy, svc)
+	err = cleanUp(ctx, mirrorDeploy, svc)
 	g.Expect(err).ShouldNot(HaveOccurred())
 
 	g.Eventually(func() error {
-		return e2eTest.tenantClient.Get(context.TODO(), client.ObjectKeyFromObject(svc), svc)
+		return e2eTest.tenantClient.Get(ctx, client.ObjectKeyFromObject(svc), svc)
 	}, "2m", "5s").ShouldNot(BeNil())
 }
 
-func TestLoadbalancerHTTPForwardFor(t *testing.T) {
```

Comment:
>[!NOTE]
The following test was added three years ago, but after catching up on an internal Slack discussion about not returning x-forward-for, I decided to remove it this time.
https://github.com/civo/civo-cloud-controller-manager/pull/8

Reply:
agreed that the current test is out of date and can be removed

We now support this via a proxy_protocol flag

https://www.civo.com/docs/kubernetes/load-balancers#proxy-protocol

we should either add a test here, or add and link an issue for a future addition of a test for this functionally. Your choice

Reply:
Thank you for your comment :pray: 
I'll go ahead and add a test for this functionality in the current PR to ensure it's covered.

Reply:
@johndietz Thank you for creating issue! :pray: